test/built-ins/ShadowRealm/prototype/importValue/import-value_FIXTURE_throws.js

```diff
@@ -0,0 +1,4 @@
+// Copyright (C) 2021 Chengzhong Wu. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+throw new Error('foobar');
```

Comment:
All good in the tests! I don't have a good suggestion but I hope this test never changes to throwing an instance of TypeError so the importValue rejection is better captured and distinguished.